doc/source/_toc.yml

```diff
@@ -215,6 +215,7 @@ parts:
           - file: rllib/user-guides
             sections:
               - file: rllib/rllib-models
+              - file: rllib/rllib-checkpoints-and-exports
```

Comment:
@sven1977 this doc does not exist, maybe you renamed the file?

Reply:
done